AppCenterDataStorage/AppCenterDataStorageTests/MSDataStoreTests.m

```diff
@@ -171,6 +173,155 @@ - (void)testApplyEnabledStateWorks {
   XCTAssertEqual(enabledCount, 1);
 }
 
+- (void)testReadWithPartitionDisabled {
```

Comment:
What is "PartitionDisabled"? Should it be "testReadWhenDataModuleDisabled"?

If you rename this test, please rename other tests that you added below.

Reply:
Addressed myself.